hamocc/mo_control_bgc.F90

```diff
@@ -18,15 +18,15 @@
 
 module mo_control_bgc
 
```

Comment:
Hi Jörg, would you mind adding comments, in which namelist the logicals can be changed? - I think it would be good for understanding their role and the function for HAMOCC. 

I'll have a more detailed look into all your changes latest on Tuesday afternoon.